app/models/trainee.rb

```diff
@@ -7,6 +7,14 @@ class Trainee < ApplicationRecord
 
   belongs_to :provider
   belongs_to :apply_application, optional: true
+
+  belongs_to :dttp_trainee,
```

Comment:
Do we need these associations (and the one on `Provider`)?

Reply:
We are using this in the `Degrees::CreateFromDttp` service
https://github.com/DFE-Digital/register-trainee-teachers/pull/1787/files#diff-a371854f759fef32317a0082d00c4c00620190a62761f4659f9ef869e15cbea9R11